cloud-native-runtimes/install-cnrt.hbs.md

```diff
@@ -65,41 +65,37 @@ To install Cloud Native Runtimes:
 
         ```console
         ---
-        # if deploying on a local cluster such as Kind. Otherwise, you can remove this field
-        provider: local
+        domain_name: example.com
+        ingress:
+        external:
+            namespace: tanzu-system-ingress
+        internal:
+            namespace: tanzu-system-ingress
         ```
 
         >**Note:** For most installations, you can leave the `cnr-values.yaml` empty, and use the default values.
 
         If you are running on a single-node cluster, such as kind or minikube, set the `provider: local`
```

Comment:
I just thought about this, but I think maybe we could add a deprecation notice here as well saying that `provider: local` will be removed in the next version or CNR *OR* just change this sentence to use to `lite.enable: true`

Reply:
Do we have any deprecation notices in our own CNR docs? I think I might go for lite.enable: true here